be/src/Project/Services/Ordering/Ordering.Api/Services/OrdersService.cs

```diff
@@ -607,6 +607,38 @@ public async Task<OrderServiceModel> SaveOrderStatusAsync(UpdateOrderStatusServi
 
                     orderItem.LastOrderItemStatusChangeId = newOrderItemStatusChange.Id;
                 }
+
+                await _mailingService.SendTemplateAsync(new TemplateEmail
```

Comment:
**question:** Isn't there a check missing that will send email only when the status has been set to cancelled? I'm concerned that it would be triggered for any status change

Reply:
I added email notifications when `serviceModel.OrderStatusId == OrderStatusesConstants.CanceledId`. You can check line 595 for the condition.

Reply:
if it is a change from the client's side, it is not necessary because the client can only set the status to 'Cancelled', but if it is a change from the admin's side, it seems that there should be a check here because he can change the status to something other than 'Cancelled'